### PR TITLE
[CLI Config Parity] fix error to handle negative numbers in gcsfuse arguments

### PR DESCRIPTION
**Pull request by ashmeenkaur**
### Description
To maintain backward compatibility, we're adding a "-" prefix to all GCSFuse arguments, aligning with cobra/viper's use of POSIX-compliant flags (which require a "--" prefix). However, this led to a bug where negative numbers were also receiving the "-" prefix, as we hadn't explicitly accounted for this scenario. The fix involves implementing a check to bypass negative numbers when adding prefixes.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Manually verified '--kernel-list-cache-ttl-secs -1` flag is working now.
2. Unit tests - Added
3. Integration tests - NA

### CODE REVIEW

**Review by ankitaluthra1 — APPROVED** [review]

File: cfg/config_util_test.go [hunk 1]
```diff
@@ -105,3 +105,25 @@ func Test_ListCacheTtlSecsToDuration_InvalidCall(t *testing.T) {
 	// Calling with invalid argument to trigger panic.
 	ListCacheTTLSecsToDuration(-3)
 }
+
+func TestIsNegativeNumber(t *testing.T) {
```
Comment (sethiay) [inline cfg/config_util_test.go:109]: could you please also write test case for --flag=postitive & --flag=negative number ? Basically try with = instead of " ".

Comment (ashmeenkaur) [reply cfg/config_util_test.go:109]: Added to main_test.go. Please take another look.

**Review by sethiay — APPROVED** [review]
